avatars/AnimationMapping.js

```diff
@@ -1,72 +1,74 @@
 class AnimationMapping {
-  constructor(animationTrackName, boneName, isTop, isPosition) {
+  constructor(animationTrackName, boneName, isTop, isPosition, isArm, isCombo) {
     this.animationTrackName = animationTrackName;
     this.boneName = boneName;
     this.isTop = isTop;
+    this.isArm = isArm;
+    this.isCombo = isCombo;
```

Comment:
`IsCombo` is not a very good name for this. Combo of what?

Reply:
`isCombo` bones are arms with Chest and UpperChest. I feel hard to name so named it `isCombo` because of mainly used for combo animation.

Used `isArm` instead, no need `isCombo` now, they are no very big differences. https://github.com/webaverse/app/pull/2791/commits/2f87848f5ea98c59458af068775dc12796a543d6